src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1434,11 +1478,17 @@
 helps['network public-ip delete'] = """
     type: command
     short-summary: Delete a public IP address.
+    long-summary: To learn more about how to delete a public IP address visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-public-ip-address#view-change-settings-for-or-delete-a-public-ip-address
+    examples:
+        - name: Delete a network interface
+          text: >
+            az network public-ip delete -n MyNic -g MyResourceGroup
```

Comment:
This example if for public-ip delete but it's written for a NIC.  That being said, this probably doesn't even warrant an example because the command usage is trivial.

Reply:
Addressed.